main.go

```diff
@@ -20,15 +20,6 @@ func main() {
 	cfp := flag.String("c", "./gerberos.toml", "Path to TOML configuration file")
 	flag.Parse()
 
-	// Already running instance?
```

Comment:
Is there a replacement or do we just drop this feature?
I don't mind if we drop it

Reply:
Let's drop it. systemd (or similar) will prevent accidental multiple instances.